From fc850ce623c24b5eb3066917599956940fc0fda3 Mon Sep 17 00:00:00 2001
From: Nicolas Goaziou <mail@nicolasgoaziou.fr>
Date: Thu, 26 Feb 2015 21:39:05 +0100
Subject: [PATCH 26/26] ox-odt: Small refactoring

* lisp/ox-odt.el (org-odt--translate-latex-fragments): Small
  refactoring.
---
 lisp/ox-odt.el | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/lisp/ox-odt.el b/lisp/ox-odt.el
index ec2be870c..386e89426 100644
--- a/lisp/ox-odt.el
+++ b/lisp/ox-odt.el
@@ -3801,7 +3801,8 @@ contextual information."
 				 (file-name-nondirectory input-file))))
 		 (display-msg
 		  (case processing-type
-		    ((dvipng imagemagick) (format "Creating LaTeX Image %d..." count))
+		    ((dvipng imagemagick)
+		     (format "Creating LaTeX Image %d..." count))
 		    (mathml (format "Creating MathML snippet %d..." count))))
 		 ;; Get an Org-style link to PNG image or the MathML
 		 ;; file.
@@ -3819,10 +3820,10 @@ contextual information."
 	    (when org-link
 	      ;; Conversion succeeded.  Parse above Org-style link to
 	      ;; a `link' object.
-	      (let* ((link (org-element-map
-			       (org-element-parse-secondary-string
-				org-link '(paragraph link))
-			       'link #'identity info t))
+	      (let* ((link
+		      (org-element-map
+			  (org-element-parse-secondary-string org-link '(link))
+			  'link #'identity info t))
 		     (replacement
 		      (case (org-element-type latex-*)
 			;; Case 1: LaTeX environment.  Mimic
-- 
2.13.2

